src/components/PokemonList.tsx

```diff
@@ -48,6 +51,11 @@ const PokemonList = () => {
                   bg="#d4d4d4"
                   borderColor="737373"
                   key={item.id}>
+                  <Image
+                    source={{uri: item.img}}
+                    style={{width: 90, height: 90}}
```

Comment:
probably want to not use hardcoded size values in the future, for now its fine https://github.com/rdmulford/pokerus/issues/20